aspnetcore/fundamentals/native-aot.md

```diff
@@ -246,6 +246,10 @@ A parameter on the delegate that isn't bound to the body and does ***not*** need
 
 :::code language="csharp" source="~/fundamentals/aot/samples/Todo.cs" id="snippet_1":::
 
+### C# 12 Interceptors
+
+The Request Delegate Generator uses the new [C# 12 interceptors compiler feature](/dotnet/csharp/whats-new/csharp-12) to support intercepting calls to minimal API’s [Map](xref:Microsoft.AspNetCore.Builder.EndpointRouteBuilderExtensions) action methods with statically generated variants at runtime. The use of interceptors results in increased startup performance for apps compiled with `PublishAot`.
```

Comment:
To me "statically generated variants at runtime" reads a bit unlucky, i.e. as these variants are generated at runtime.
Can you please rephrase that? Unfortunately I don't have a proper suggestion for it.

Reply:
@mitchdenny please use the suggestion mechanism

Reply:
Here is how I would write it:

> The Request Delegate Generator uses the new C# 12 interceptors compiler feature to support intercepting calls to minimal APIs Map methods. Map method calls are replaced with code generated calls at compile time. The compiler generated implementations are compatible with native AOT and results in increased startup performance for apps published with the PublishAot property enabled.